packages/electrumAnalyzer/electrumAnalyzer.0.3.4/opam

```diff
@@ -0,0 +1,24 @@
+opam-version: "1.2"
+maintainer: "David Chemouil <david.chemouil+electrum@onera.fr>"
+authors: ["David Chemouil" "Julien Brunel" "Denis Kuperberg"]
+homepage: "https://forge.onera.fr/projects/electrum"
+bug-reports: "https://forge.onera.fr/projects/electrum"
+license: "GPL-3+"
+dev-repo: "https://git.onera.fr/electrum.git"
+build: [
+  ["./configure" "--prefix=%{prefix}%"]
+  [make]
+]
+install: [make "install"]
+remove: [
+  ["rm" "-f" "%{bin}%/electrumAnalyzer"] {os = "darwin" | os = "linux"}
+]
+depends: [
+  "batteries" {build}
+  ("cmdliner" {build & = "0.9.7"} | "cmdliner" {build & = "0.9.6"} |
+   "cmdliner" {build & = "0.9.5"})
+  "menhir" {build & >= "20150720"}
+  "ocamlfind" {build & >= "1.5.5"}
+  "pprint" {build}
```

Comment:
@Chris00 I did try to use the result of oasis2opam as an inspiration but eventually built the opam file by myself (helped by `opam pin edit`). Indeed, I wasn't sure how it fits with the use of `opam-publish prepare` as prescribed in the Opam packaging documentation. E.g. it creates a `files` directory.

By the way, my Opam "remove" configuration currently calls `rm -f` as the `ocamlfind remove` command inserted by `opam pin` in the opam template didn't remove the installed file (the electrumAnalyzer binary) when I called `opam remove electrumAnalyzer`. I would gladly get rid of this `rm -f` as it is certainly not working on OS'es other than GNU/Linux or Darwin. Any advice?